gen/util_for_test.go

```diff
@@ -37,3 +38,13 @@ type thriftType interface {
 	ToWire() (wire.Value, error)
 	FromWire(wire.Value) error
 }
+
+// streamingThriftType is implemented by all generated types that know how to
+// write and read themselves to the Thrift Protocol, skipping over the
+// intermediary wire.Type
+type streamingThriftType interface {
+	thriftType
+
+	Encode(stream.Writer) error
+	//Decode(stream.Reader) error
```

Comment:
Decode?

Reply:
Commented out for now until we can merge #496 and this PR together.